3p/integration.js

```diff
@@ -219,7 +219,6 @@ window.draw3p = function(opt_configCallback, opt_allowed3pTypes,
 
     // This only actually works for ads.
     const initialIntersection = window.context.initialIntersection;
-    delete window.context.initialIntersection;
```

Comment:
Can we leave the inititalIntersection property on the context? For our use case we will need to effectively recreate it pretty much as the first step, so might as well keep it:)